projects/perception/heart_anomaly_detection/README.MD

```diff
@@ -0,0 +1,17 @@
+# Heart Anomaly Detection
+
+This folder contains a sample application for heart anomaly detection. It can be used for both evaluation on AF dataset, as well as inference on a single ECG timeseries provided as .npy array.
```

Comment:
```suggestion
This folder contains a sample application for heart anomaly detection.
It can be used for both evaluation on AF dataset, as well as inference on a single ECG timeseries provided as .npy array.
```